test/parallel/test-assert.js

```diff
@@ -183,7 +183,7 @@ assert.doesNotThrow(makeBlock(a.deepEqual, a1, a2));
 
 // having an identical prototype property
 const nbRoot = {
-  toString: function() { return `${this.first} ${this.last}`; }
+  toString: () => { return `${this.first} ${this.last}`; }
```

Comment:
The same.

Reply:
Thank you for your review.
I was mistaken.

I'll fix this problem with like:

```js
  toString() { return `${this.first} ${this.last}`; }
```